Build error in RobotiqHandPlugin.h

I get a couple of these when trying to build the package. They all appear to be in the RobotiqHandPlugin.h file.


src/robotiq/robotiq_s_model_articulated_gazebo_plugins/include/robotiq_s_model_articulated_gazebo_plugins/RobotiqHandPlugin.h:176:47: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double RobotiqHandPlugin::VelTolerance’ of non-integral type [-fpermissive]
   private: static const double VelTolerance = 0.002;

--- a/robotiq_s_model_articulated_gazebo_plugins/CMakeLists.txt
+++ b/robotiq_s_model_articulated_gazebo_plugins/CMakeLists.txt
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(robotiq_s_model_articulated_gazebo_plugins)
+add_compile_options(-std=c++11)
 
 find_package(catkin REQUIRED COMPONENTS std_msgs gazebo_plugins actionlib tf image_transport control_msgs trajectory_msgs geometry_msgs sensor_msgs roscpp gazebo_ros robotiq_s_model_articulated_msgs)
 

--- a/robotiq_s_model_articulated_gazebo_plugins/include/robotiq_s_model_articulated_gazebo_plugins/RobotiqHandPlugin.h
+++ b/robotiq_s_model_articulated_gazebo_plugins/include/robotiq_s_model_articulated_gazebo_plugins/RobotiqHandPlugin.h
@@ -173,18 +173,18 @@ class RobotiqHandPlugin : public gazebo::ModelPlugin
 
   /// \brief Velocity tolerance. Below this value we assume that the joint is
   /// stopped (rad/s).
-  private: static const double VelTolerance = 0.002;
+  private: static constexpr double VelTolerance = 0.002;
 
   /// \brief Position tolerance. If the difference between target position and
   /// current position is within this value we'll conclude that the joint
   /// reached its target (rad).
-  private: static const double PoseTolerance = 0.002;
+  private: static constexpr double PoseTolerance = 0.002;
 
   /// \brief Min. joint speed (rad/s). Finger is 125mm and tip speed is 22mm/s.
-  private: static const double MinVelocity = 0.176;
+  private: static constexpr double MinVelocity = 0.176;
 
   /// \brief Max. joint speed (rad/s). Finger is 125mm and tip speed is 110mm/s.
-  private: static const double MaxVelocity = 0.88;
+  private: static constexpr double MaxVelocity = 0.88;
 
   /// \brief Default topic name for sending control updates to the left hand.
   private: static const std::string DefaultLeftTopicCommand;

